roles/openshift_node/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
 
 - name: Install sdn-ovs package
   action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-sdn-ovs{{ openshift_pkg_version | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
-  when: openshift.common.use_openshift_sdn and not openshift.common.is_containerized | bool
+  when: openshift.common.use_openshift_sdn and not openshift.common.use_flannel | bool and not openshift.common.is_containerized | bool
```

Comment:
@detiber We're sanity checking the values but we aren't saying, "this variable needs to be set".